cmd/control_loop/controller/controller_test.go

```diff
@@ -28,48 +26,34 @@ func unavailable(err error) bool {
 	return st.Code() == codes.Unavailable
 }
 
-func check(t *testing.T, message string, err error) {
-	if unavailable(err) {
-		t.Logf("Unable to connect to registry server. Is it running?")
-		t.FailNow()
-	}
-	if err != nil {
-		t.Errorf(message, err.Error())
-	}
-}
-
 func checkActions(expectedActions, actions []string) string {
 	opts := cmpopts.SortSlices(func(a, b string) bool { return a < b })
 	diff := cmp.Diff(expectedActions, actions, opts)
 	return diff
 
 }
```

Comment:
Knowing that actions are checked with order ignored seems like useful information to have in the test rather than hidden in a helper function. Consider creating a file level variable `var sortStrings := cmpopts.SortSlices(func(a, b string) bool { return a < b })` that can remove the duplication while keeping tests descriptive.

Reply:
Done